From 0a34645e1bfa7181ed384e2e9ef0920445f41267 Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Thu, 2 Dec 2021 08:31:09 -0500
Subject: [PATCH] Drop the setuptools_git_ls_files dependency

This dependency makes sense upstream, but we do not need it (and it is
not packaged) in Fedora.
---
 pyproject.toml | 1 -
 setup.py       | 2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 8b63d75..591a65e 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -3,7 +3,6 @@ requires = [
     "setuptools",
     "wheel",
     "setuptools_scm",
-    "setuptools_git_ls_files",
     "cython",
 ]
 
diff --git a/setup.py b/setup.py
index 3334b82..9e692e7 100644
--- a/setup.py
+++ b/setup.py
@@ -152,7 +152,7 @@ setup_params = dict(
         'build_ext': custom_build_ext,
     },
     setup_requires=(
-        ["setuptools_scm", "setuptools_git_ls_files"] + pytest_runner + wheel
+        ["setuptools_scm"] + pytest_runner + wheel
     ),
     tests_require=[
         'pytest>=2.8',
-- 
2.33.1

